iframes/iframe-integration.js

```diff
@@ -35,4 +35,15 @@ for (const key in iframeAttributes) {
 
 iframeResize({}, iframe)
 
-script.before(iframe)
+if (script.parentElement.tagName === "HEAD") {
+  const htmlDocument = script.parentElement.parentElement
+  const children = htmlDocument.childNodes
+  for (var i = 0; i < children.length; i++) {
+    if (children[i].tagName === "BODY") {
+      children[i].appendChild(iframe)
+      break
+    }
+  }
```

Comment:
Je crois qu'on peut simplifier en utilisant directement la propriété body du document

```suggestion
  const body = htmlDocument.body
  if (body) {
    body.appendChild(iframe)
  } else {
    console.error("Impossible de trouver l'élément <body> dans le document HTML.")
  }
```

Reply:
Non je ne pense pas, peut-être que le nom de la variable n'est pas bon. `htmlNode` est surement mieux car en testant dans une console `temp0.parentElement.parentElement.body` qui est `undefined` pour un `html` node.

Reply:
Oui, probablement. J'ai fait un Wix de mon côté et j'ai bien constaté le problème